pulsar-client-api/src/main/java/org/apache/pulsar/client/api/PulsarClient.java

```diff
@@ -281,7 +281,8 @@ static ClientBuilder builder() {
      * Perform immediate shutdown of PulsarClient.
      *
      * <p>Release all the resources and close all the producer, consumer and reader instances without waiting
-     * for ongoing operations to complete.
+     * for ongoing operations to complete. Resources passed from the outside like EventLoopGroup or ConnectionPool
+     * will not be closed.
```

Comment:
`EventLoopGroup` and `ConnectionPool` classes are not part of the public API (even though they can be passed to a `PulsarClientImpl` constructor, which is not public), so they should not be mentioned here.

Reply:
@merlimat @MarvinCai was suggesting to document it so I added it there. Any alternative or we don't need to mention that at all?

Reply:
PulsarClientImpl's constructors are actually public(probably shouldn't be as we don't expect user direct create one themself?) 
@abhilashmandaliya I was thinking about adding doc on PulsarClientImpl's constructor, but as it not intended to be used directly by user. And seems the ClientBuilder doesn't support pass in eventloopgroup and cnxpool for now, it doesn't make much sense to tell user about the internal of pulsarclient, we probably don't need that doc for now, and if in the future we support pass in eventloopgroup/cnxpool we can add them then.

Reply:
alright. I have reverted the java doc comment. Hope tests pass this time :crossed_fingers: